doc/devel/index.rst

```diff
@@ -77,9 +84,7 @@ New contributors
 
       :octicon:`question;1em;sd-text-info` :ref:`Where should I ask questions? <get_connected>`
 
-      :octicon:`issue-opened;1em;sd-text-info` :ref:`What are "good-first-issues"? <new_contributors>`
-
-      :octicon:`git-pull-request;1em;sd-text-info` :ref:`How do I claim an issue? <managing_issues_prs>`
+      :octicon:`git-pull-request;1em;sd-text-info` :ref:`How do I choose an issue? <managing_issues_prs>`
```

Comment:
I find this slightly off: I believe many contributions start because people have a certain idea or problem they are interested in. I assume it's rather uncommon that people say "Oh I want to work on Matplotlib. What can I do?"

I have no good alternative suggestion yet. Just saying the current phrasing is tailored to a rather edge case.

Reply:
> "how do I work on issues" maybe?

Yes, or "How to work on issues".